src/capi.rs

```diff
@@ -441,23 +441,30 @@ pub unsafe extern fn rav1e_config_parse_int(
   }
 }
 
+fn context_new_internal(cfg: &rav1e::Config) -> Result<Context, rav1e::EncoderStatus> {
```

Comment:
Is there a reason this needs to be a separate function?

Reply:
You can `map()` and `unwrap_or()` to the same effect but it would be less readable.